htdocs/opensurvey/card.php

```diff
@@ -404,7 +404,7 @@
 // Comment list
 $comments = $object->getComments();
 
-if ($comments) {
+if (!empty($comments)) {
```

Comment:
Note: Because, just after there is a "foreach", a better test would be
`if (!empty($comments)) && is_array($comments)) {`